sdcm/utils/common.py

```diff
@@ -1148,38 +1148,53 @@ def filter_gce_instances_by_type(instances):
     return filtered_instances
 
 
-BUILDERS = [
-    {
-        "name": "aws-eu-builder-2",
-        "public_ip": "34.249.55.96",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-3",
-        "public_ip": "34.248.136.231",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-4",
-        "public_ip": "99.81.48.181",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-5",
-        "public_ip": "18.200.171.172",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    },
-    {
-        "name": "aws-eu-builder-6",
-        "public_ip": "99.81.50.63",
-        "user": "jenkins",
-        "key_file": os.path.expanduser("~/.ssh/scylla-qa-ec2")
-    }
-]
+SSH_KEY_DIR = "~/.ssh"
+SSH_KEY_AWS_DEFAULT = "scylla-qa-ec2"
+SSH_KEY_GCE_DEFAULT = "scylla-test"
+
+
+def get_aws_builders():
+    builders = []
+    ssh_key_path = os.path.join(SSH_KEY_DIR, SSH_KEY_AWS_DEFAULT)
+
+    aws_builders = list_instances_aws(tags_dict={
+        "NodeType": "Builder"
+    })
+
+    for aws_builder in aws_builders:
+        builder_name = [tag["Value"] for tag in aws_builder["Tags"] if tag["Key"] == "Name"][0]
+        builders.append({
+            "public_ip": aws_builder["PublicIpAddress"],
+            "name": builder_name,
+            "user": "jenkins",
+            "key_file": os.path.expanduser(ssh_key_path)
+        })
+
+    return builders
+
+
+def get_gce_builders():
+    builders = []
+    ssh_key_path = os.path.join(SSH_KEY_DIR, SSH_KEY_GCE_DEFAULT)
+
+    gce_builders = list_instances_gce(tags_dict={"nodetype": "builder"})
+
+    for gce_builder in gce_builders:
+        builders.append({
+            "public_ip": gce_builder.public_ips[0],
+            "name": gce_builder.name,
+            "user": "scylla-test",
```

Comment:
builder user is `jenkins`.
 `scylla-test` is used by SCT

Reply:
i will recheck, but i can't get access to gce instances with jenkins user and gce key

Reply:
@bentsi  can't get access with jenkins ans scylla-test key
```
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-test jenkins@35.229.76.4
jenkins@35.229.76.4: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-qa-ec2 ### jenkins@35.229.76.4
jenkins@35.229.76.4: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-test jenkins@35.229.76.4
jenkins@35.229.76.4: Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
abykov@ububuntu:~/Projects/scylla-cluster-tests$ ssh -i ~/.ssh/scylla-test scylla-test@35.229.76.4
Last login: Fri Nov  8 22:11:13 2019 from 50.233.106.94
[scylla-test@public-jenkins-builder4-qavpc ~]$ exit
logout
Connection to 35.229.76.4 closed.
```

Reply:
use scylla-test user

Reply:
i.e. the jenkins for some reason on gce has it's own key

Reply:
For some reason, it works only for builder 1..., let's keep it like you implemented.